src/dotnet-user-secrets/Internal/InitCommand.cs

```diff
@@ -0,0 +1,99 @@
+using Microsoft.Extensions.CommandLineUtils;
```

Comment:
Two code formatting nit-picks and all new *.cs files:
* Can you add the copyright header?
* Can you sort using with System.* namespaces first? I wish VS enforced this better... :-/ https://github.com/aspnet/DotNetTools/blob/581b3160f422180b3411d84b4458c10128c8cacb/.editorconfig#L12-L14